components/console/helpers/progressbar.rst

```diff
@@ -337,13 +337,23 @@ display that are not available in the list of built-in placeholders, you can
 create your own. Let's see how you can create a ``remaining_steps`` placeholder
 that displays the number of remaining steps::
 
+    // This definition is globally registered for all ProgressBar instances
     ProgressBar::setPlaceholderFormatterDefinition(
         'remaining_steps',
         function (ProgressBar $progressBar, OutputInterface $output) {
             return $progressBar->getMaxSteps() - $progressBar->getProgress();
         }
     );
 
+It is also possible to set a placeholder formatter per ProgressBar instance
+with the ``setPlaceholderFormatter`` method::
+
+    $progressBar = new ProgressBar($output, 3, 0);
+    $progressBar->setFormat('%countdown% [%bar%]');
+    $progressBar->setPlaceholderFormatter('countdown', function (ProgressBar $progressBar) {
+        return $progressBar->getMaxSteps() - $progressBar->getProgress();
+    });
+
```

Comment:
We should probably add

```
.. versionadded:: 6.3

    The ``setPlaceholderFormatter()`` method was introduced in Symfony 6.3.
```

Reply:
I thought of this, but wasn't sure! Do you know if there is any "rule" to it, when to add it or not?

Reply:
@alexandre-daubois we add the `versionadded` directive for any new option, method, class, feature, etc. added to Symfony. We also have the `deprecated` directive to do the opposite.